client/src/App.tsx

```diff
@@ -6,71 +6,46 @@ import { Registration } from './pages/Registration';
 import { ProductDetails } from './pages/ProductDetails';
 import { NotFound } from './pages/NotFound';
 import { Home } from './pages/Home';
+import { CartContext } from './components/CartContext';
+import { CartProduct } from './components/CartContext';
```

Comment:
Usually you would only list each file once and combine the imports:
`import { CartContext, CartProduct } from './components/CartContext';`